src/python/setup.py

```diff
@@ -14,6 +14,8 @@
 
 PACKAGE_NAME = "turicreate"
 VERSION = "6.2"  # {{VERSION_STRING}}
+NON_MINIMAL_LIST = ["tensorflow", "resampy"]
```

Comment:
Let's get pandas in here too?  Is that possible?

Reply:
let's do that in the next phase (another PR on its way)